### ISSUE TITLE
[CoE Starter Kit - BUG] Power Apps portals status is not updated

### ISSUE THREAD

**Issue by bhavikj65** [microsoft/coe-starter-kit#2725]
### Describe the issue

Power Apps portals which are deleted from the environments are still displayed as Active in CoE Dashboard. 

### Expected Behavior

Deleted portals should be marked as deleted.

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

3.38

### What app or flow are you having the issue with?

Admin | Sync Template v3 (Portals)

### Steps To Reproduce

1. Create a Power App portal in any of the existing or new Power Platform environment.
2. Wait for CoE to populate details about this new portal in CoE Dashboard.
3. Once details of this new portal are visible in CoE, delete the portal from that environment using Power Platform Admin center.
4. After successful deletion, wait for the CoE flows and you will see the portal is still visible in CoE Dashboard. 

AB#1301

**Comment by bhavikj65** [comment]
@petepuu

**Comment by Jenefer-Monroe** [comment]
Hello. You are correct on this limitation. Today we cannot detect when a portal has been deleted but there are some upcoming changes from the product which we will utilize to fix this. Putting on the backlog as a result. 
We will validate this week to make sure there isn't something we are missing to plug this hole awaiting product change, or if someone in the community has an idea we are open to it.

**Comment by dchrist27** [comment]
Hi...I still see the 'bug' on the Power Apps Portals Creation Trend page for the Nov 2022 CoE Starter Kit.   Any update on when this bug will be addressed?

**Comment by Jenefer-Monroe** [comment]
Hello, we are blocked by a product limitation here and cannot determine that portals have been deleted until a change from them is implemented. If you delete the portal manually from your inventory though, it will not be re-added.

**Comment by dchrist27** [comment]
Hi…

I can see all of the old portals also in the Power Platform Admin view app…but I didn’t see an option to delete.    Does the inventory exist in a table?

Thanks

**Comment by Jenefer-Monroe** [comment]
Yep, you can edit and delete data right in the table now with some recent product changes. 
Go to your CoE environment > Core Components > Tables > Power Pages Site 
From here you can edit what fields are shown or select Edit to bring you to a full page view
![image](https://user-images.githubusercontent.com/89584709/216175950-ad3a9e0d-c554-4adb-96a2-5d86f7c5084e.png)

From the full page view you can find your row and delete
![image](https://user-images.githubusercontent.com/89584709/216176252-e2c762ff-0e44-4ba2-9fe1-a9f991a28f9b.png)

**Comment by dchrist27** [comment]
Perfect.  Thank you for posting.

**Comment by Jenefer-Monroe** [comment]
you bet, happy to be able to help!

**Comment by BroxCJ** [comment]
Hi, I believe this is a manual work around, but I don't really think it's a proper solution.

So, is there any expected date for this? (such a "deleted column or even directly, not shown the deleted Pages/sites).

Finally, there are some other things we are missing (such as portal owners/admin), which I think it may be another topic in another thread.

Thanks

**Comment by Jenefer-Monroe** [comment]
The CoE kit is built on top of the product, and as such bound to platform limitations. Providing this feedback to the product team through the product feedback channels will be your best bet to getting this incorporated first into the product, and then into the kit!

**Comment by BroxCJ** [comment]
Hi Jennifer, I think I understood wrongly this last answer, as I thought it may mean that this field should come from the PowerPlatform, so I opened a ticket with them, and they are answering me that this is only a COE feature.

So, at this point, as It seems I made a mistake on who I had to approach: Which could be the "Product feedback Channels" if not Msft ones? (PowerPlatform ones). Thanks!

**Comment by Jenefer-Monroe** [comment]
The request just needs to be phrased appropriately to the product team. And I dont think support is the correct channel as they only help with things that are already implemented and just not working correctly. The ask would be to the product team to say, there is no way to programmatically tell if a portal is still active as inactive ones stay listed in the product tables unmarked as deleted. If you dont have direct access to a MSFT sales person for example, then this can be done through https://powerusers.microsoft.com

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue. 
Thank you for your understanding and cooperation.
